ce/ce/artifacts/activation.ts

```diff
@@ -740,6 +746,10 @@ function generateScriptContent(kind: string, variables: Record<string, string>,
   return '';
 }
 
+function generateEnvVarDump(variables: Record<string, string>): string {
+  var contents = {"version": 1, "env": variables}
```

Comment:
I'd put the other things (aliases, properties , etc) in here too so that this doesn't have to be done again